rfcs/ambassador-program/index.md

```diff
@@ -0,0 +1,83 @@
+## GraphQL Ambassador Program
+
+The following is the beginnings of a document outlining the goals, outcomes and tasks of an Ambassador program. 
+
+### Alternative Names
+
+* Champion - already used on the technical side - Champions of spec edits
+* Evangelist - usually out of favour nowadays
+* Advocate - suggests a salaried role?
+* Leader, Star, Hero, Captain, Expert - “Star” we have already used in a recognition role
+* GraphQL Fellowship 
+
+> [!NOTE]
+> “Ambassador” seems to be readily accepted, I don't think we need to discuss the term any further.   
+
+### Ambassador Initiative Goals
+
+The overarching goal is to increase GraphQL adoption. The Ambassadors program aims to accomplish this through:
+
+* Brand advocacy - developers organically vouch for and promote GraphQL
+* Social proof - named individuals with high visibility (and a proportion from known corporate brands) as Ambassadors
+* Increased GraphQL content online and off
+* Community building - increase the number of “known faces” within GraphQL, increased approachability (addressing the concern of “lowering the barrier to join an inclusive GraphQL community” raised by the Community WG)  
+
+How can these goals be measured? 
+
+* Engagement metrics - social, chat channels, search, downloads, page views…
+* New community members (across social media comments, Discord users, local gathering attendees, working group members…) 
+* Volume of talks at conferences
+* Volume of articles written
+
+### Ambassador Tasks
+
+As a GraphQL Ambassador, you will be an important touchstone for people interested in GraphQL to learn more about it in a friendly and constructive way. 
+
+One or more of the following:
+
+* Conference & meetup talks on GraphQL
+* Local gathering organization
+* Written GraphQL content
+* Video / audio GraphQL content
+
+### Foundation Tasks
+
+* Grant available for travel expenses for approved conference & meetup talks
+* Microgrant (or honorarium) available for content creation 
+    * Where is this content hosted? 
+* Promotion of Ambassador material - conference talk videos and created content
+* Promotion of the Ambassador as a go-to GraphQL expert
+    * Website profile
+    * Longer task: biographies / interviews with each Ambassador
+    * Credly badge
+* Exclusive GraphQL swag eg Ambassador hoodie
```

Comment:
Maybe quarterly meeting with the foundation? Give the ambassador updates about the general direction and also an opportunity for them to ask direct questions?

Reply:
I think this is a great idea. In fact, there has been some discussion on building a community for GraphQL implementers (ie those who maintain the reference implementations) where the central activity is a meeting or update of some kind about what is going on with GraphQL. EG this would have been a very helpful initiative to already have for the nullability working group to gather feedback. 

I wonder if it should be like a formal presentation style, or more "office hours" style?

Reply:
Both? 15min updates from the foundation to kick things off (what are the current initiatives) + "office hours"/Q&A?